superset-frontend/stylesheets/superset.less

```diff
@@ -582,3 +582,35 @@ hr {
   background-image: url('../images/icons/error_solid_small_red.svg') !important;
   background-position: -2px center !important;
 }
+
+// Ant-d overrides since these are injected as inline styles and can't
+// be overrided in emotion
+.ant-menu-submenu.ant-menu-submenu-popup.ant-menu.ant-menu-light.ant-menu-submenu-placement-bottomLeft {
+  top: 51px !important;
+  margin-left: -2px !important;
+  @media (max-width: 767px) {
+    top: 269px !important;
+  }
+  & > .ant-menu {
+    border-radius: 0px !important;
+  }
+}
+
+.ant-menu-submenu.ant-menu-submenu-popup.ant-menu.ant-menu-light {
+  top: 51px !important;
+  border-radius: 0px !important;
+  @media (max-width: 767px) {
+    top: 269px !important;
```

Comment:
This whole block of new styles contains a lot of highly specific pixel measurements. Is this introducing fragility by pinning things in particular locations? I just want to make sure that this doesn't look all broken if something else changes, like the height of a logo or something.

Reply:
The only thing that will cause issue is when the if a user is on has all the submenu inline tabs open. The onhover dropdown might be off. Now that i think about it might make sense to turn these in submenu dropdowns as well.

> What do we need to do to get this through?

@mistercrunch Evan and were trying to figure out how to fix the overrides that we have to put in the less file. This will most likely end up as a follow up as this is not really a blocker.